wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -549,6 +553,58 @@ public static function setOfficeDetails($op, $groupID, $entityID, &$params) {
 
   }
 
+  /**
+   * This hook is called after the database write on a custom table.
+   *
+   * @param string $op
+   *   The type of operation being performed.
+   * @param string $objectName
+   *   The custom group ID.
+   * @param int $objectId
+   *   The entityID of the row in the custom table.
+   * @param object $objectRef
+   *   The parameters that were sent into the calling function.
+   */
+  public static function displayActivityDataForCollectionCamp($op, $groupID, $entityID, &$params) {
```

Comment:
The name of this function is not accurate because this function does not display anything. It saves the induction ID of the initiator of collection camp within the collection camp.

So it can be named something like: `linkInductionWithCollectionCamp`